CMakeLists.txt

```diff
@@ -51,7 +51,6 @@ endif()
 
 
 # build dependencies
-find_package(Gpgme REQUIRED)
```

Comment:
Now `cmake/modules/FindGpgme.cmake` is unused. Do we want to also remove it?

Reply:
Thank you for the notice. I removed the file.